lib/nrf_modem_lib/nrf_modem_os.c

```diff
@@ -365,6 +365,16 @@ void nrf_modem_os_application_irq_clear(void)
 	NVIC_ClearPendingIRQ(NRF_MODEM_APPLICATION_IRQ);
 }
 
+void nrf_modem_os_application_irq_disable(void)
+{
+	NVIC_DisableIRQ(NRF_MODEM_APPLICATION_IRQ);
+}
+
+void nrf_modem_os_application_irq_enable(void)
+{
+	NVIC_EnableIRQ(NRF_MODEM_APPLICATION_IRQ);
+}
+
```

Comment:
We ended up changing the modem lib implementation and these are no longer needed with the updated https://github.com/NordicPlayground/libmodem/pull/126 PR.

Reply:
Adjusted.